src/sentry/static/sentry/app/actionCreators/savedSearches.jsx

```diff
@@ -21,7 +21,13 @@ export function fetchSavedSearches(api, orgId, useOrgSavedSearches = false) {
 
   promise
     .then(resp => {
-      SavedSearchesActions.fetchSavedSearchesSuccess(resp);
+      // Add in project slugs so that we can display them in the picker bars.
+      const savedSearchList = resp.map(search => ({
+        ...search,
+        projectSlug: projectMap[search.projectId],
+      }));
```

Comment:
Would it be good to leave a TODO to remove this code when org-saved-searches is mainlined?